### PR TITLE
Fix/tabs docs

### PR DESCRIPTION
**Pull request by j-mnizhek**
## Motivation and Context

We forgot to merge design documentation, so I've brought it as a several PR to avoid dealing with conflicts.

## How has this been tested?

Manually.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by j-mnizhek** [comment]
@msereniti Michael, please check if everything is okay in the examples for tabs 🙏

**Comment by j-mnizhek** [comment]
@ilyabrower Ilya, when you have time, please check if everything is okay in the examples for tabs 🙏

File: website/docs/components/tab-line/examples/manual_tab_activation.tsx [hunk 1]
```diff
@@ -2,31 +2,28 @@ import React from 'react';
...
-        <TabLine.Item value={4} disabled>
-          Disabled
+          Birds
         </TabLine.Item>
       </TabLine>
       {
         [
-          <div id='tab-panel-1' role='tabpanel' aria-labelledby='tab-label-1' tabIndex={-1}>
-            <h3>Overview</h3>
+          <div id='tab-panel-0' role='tabpanel' aria-labelledby='tab-label-1' tabIndex={-1}>
```
Comment (ilyabrower) [inline website/docs/components/tab-line/examples/manual_tab_activation.tsx:22]: `id='tab-panel-1'` should be here

File: website/docs/components/tab-panel/examples/automatic_tab_activation.tsx [hunk 2]
```diff
@@ -2,31 +2,28 @@ import React from 'react';
...
-        <TabPanel.Item value={4} disabled>
-          Disabled
+          Birds
         </TabPanel.Item>
       </TabPanel>
       {
         [
-          <div id='tab-panel-1' role='tabpanel' aria-labelledby='tab-label-1' tabIndex={-1}>
-            <h3>Overview</h3>
+          <div id='tab-panel-0' role='tabpanel' aria-labelledby='tab-label-1' tabIndex={-1}>
```
Comment (ilyabrower) [inline website/docs/components/tab-panel/examples/automatic_tab_activation.tsx:22]: and here too - `id='tab-panel-1'`

File: website/docs/components/tab-panel/examples/manual_tab_activation.tsx [hunk 3]
```diff
@@ -1,35 +1,60 @@
...
+        <TabPanel.Item value={3} aria-controls='tab-panel-3'>
+          Birds
         </TabPanel.Item>
-      </Tooltip>
-      <TabPanel.Item value={4}>Statistics</TabPanel.Item>
-    </TabPanel>
+      </TabPanel>
+      {
+        [
+          <div id='tab-panel-0' role='tabpanel' aria-labelledby='tab-label-1' tabIndex={-1}>
```
Comment (ilyabrower) [inline website/docs/components/tab-panel/examples/manual_tab_activation.tsx:22]: and here

**Pull request merged**